src/service/binding.ts

```diff
@@ -194,11 +213,20 @@ export type BindingClassPrototype<Binding> = {
     : never;
 };
 
+/**
+ * an alias
+ */
 type ServiceBindingClass<T> = new (
```

Comment:
The `ServiceBindingClass ` is not just an alias. the `type BindingClassPrototype ` would create an object like this

```ts
{
  transfer: (payload, privateKey? ) => Transaction | Receipt
}
```

and the  `type ServiceBindingClass ` would create a class(not an instance)